guides/common/modules/proc_converting-hosts-with-convert2rhel.adoc

```diff
@@ -0,0 +1,23 @@
+[id="converting-hosts-with-convert2rhel_{context}"]
+
+= Converting Hosts with Convert2RHEL
+
+Use the following procedure to perform the conversion.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* *>* *All Hosts* and select the target hosts you want to convert.
+You can use the *Search* field to filter the host list.
+. From the *Select Action* list, select Schedule Remote Job.
+. On the *Job Invocation* page, define the main job settings.
```

Comment:
this and the line below should be swapped, first they must select the right job category only then they can set all job settings as the first step slightly modifies available settings

Reply:
Can I check which two lines you mean here, Marek?

Reply:
"On the *Job Invocation* page, define the main job settings." and after the previous change the following two lines "In the *Job category* field, select Convert to RHEL.
In the *Job template* field, verify that the Job Template is set to `Convert to RHEL`."

The "On the ..." should be 3rd of them, in other words it should go like this

1. In the *Job category* field, select Convert to RHEL.
2. In the *Job template* field, verify that the Job Template is set to `Convert to RHEL`.
3. On the *Job Invocation* page, define the main job settings.

Reply:
Fixed